goodtablesio/utils/jobconf.py

```diff
@@ -119,25 +59,75 @@ def make_validation_conf(job_files, job_conf, job_base=None):
                         and job_base):
                     item['schema'] = '/'.join(
                         [job_base, item['schema']])
-                validation_conf['files'].append(item)
-
-    # Copy settings
+                validation_conf['source'].append(item)
+
+    # Datapackages
+    if 'datapackages' in job_conf:
```

Comment:
So in theory we could have configurations with both `files`  and `datapackages`, wouldn't these cause issues? eg files could get validated twice? Perhaps is best if both are exclusive, ie one or the other?